### DISCUSSION TITLE
LilyGO T5 4.7 inch E-paper Display and LVGL

### DISCUSSION THREAD

**Discussion by ianmxyz** [arendst/Tasmota#15930] [Q&A] [answered]
I managed to compile firmware for this and it works with the basic display commands. However I'd like to use LVGL/Berry to display graphics etc. When I try to start LVGL I get this in the Berry console.

   lv.start()
BRY: Exception> 'syntax_error' - input:1: 'lv' undeclared (first use in this function)

I'm wondering where I'm going wrong? Has anyone else managed to get this to work?

**Comment by s-hadinger** [comment, marked as answer]
You need to compile with LVGL enabled. Preferably use tasmota32-lvgl.

**Reply by ianmxyz** [reply]
I'll give it a try when I've got some time and report back.

**Comment by Jason2866** [comment]
@s-hadinger I must have missed the LVGL support for the LilyGo T5 ePaper Display. Since when is LVGL supported with this Display?

**Reply by s-hadinger** [reply]
I didn't say there is LVGL support for LilyGo T5 ePaper :-). But if the universal display driver supports it, it should work.
